apps/native/app/README.md

```diff
@@ -10,56 +10,55 @@ bundle install
 npx pod-install
 ```
 
-### 2a. Building for iOS
+## 2a. Building for iOS
 
 ```bash
 yarn run ios
 ```
 
-### 2b. Building for Android
+## 2b. Building for Android
 
-Follow documentation for setting up android [here](https://reactnative.dev/docs/environment-setup?platform=android&guide=native).
+1. Follow [Android setup documentation](https://reactnative.dev/docs/environment-setup?platform=android&guide=native).
+2. Open the project in Android Studio.
+3. Ensure `local.properties` and `google-services.json` are added:
 
-Open the project in android studio and then local.properties should be created, if you want to skip that, make sure to:
+   - `local.properties` in `/apps/native/app/android`
+   - `google-services.json` in `/apps/native/app/android/app`
 
-Add `local.properties` file to `/apps/native/app/android`
+4. Open an Android emulator or connect a device.
 
-Add `google-services.json` file to `/apps/native/app/android/app`
+Run the following in `/apps/native/app/android`:
 
-Open up an android emulator or connect a physical device
-
-in `/apps/native/app/android` run the following commands:
-
-```ts
-// Build the project
+```bash
+# Build the project
 ./gradlew assembleDebug
 ```
 
-If using an **emulator** do the following:
+### For an Emulator
```

Comment:
This is actually more of an America vs Europe thing instead of a English vs Icelandic thing. Most of the standards are from American universities and papers. Sentence case is more of a norm in Iceland/Europe. Just heck out gov.uk, bbc.com and others.

I really don't want to argue about this in review comments. We have been using sentence case. Changing that is a bigger decision/discussion.